src/test/java/com/jcraft/jsch/KeyPairTest.java

```diff
@@ -56,21 +55,20 @@ void loadKey(String path, String password, String publicKeyType) throws URISynta
         final String prvkey = Paths.get(ClassLoader.getSystemResource(path).toURI()).toFile().getAbsolutePath();
         assertTrue(new File(prvkey).exists());
         assertDoesNotThrow(() -> {
-                    if (null != password) {
-                        jSch.addIdentity(prvkey, password);
-                    } else {
-                        jSch.addIdentity(prvkey);
-                    }
-                }
-        );
+            if (null != password) {
+                jSch.addIdentity(prvkey, password);
+            } else {
+                jSch.addIdentity(prvkey);
+            }
+        });
     }
 
     @Test
     void genKeypair() {
         final JSch jSch = new JSch();
         assertDoesNotThrow(() -> {
             KeyPair kpair = KeyPair.genKeyPair(jSch, KeyPair.RSA, 1024);
-            kpair.writePrivateKey("my-private-key");
+            kpair.writePrivateKey(System.getProperty("java.io.tmpdir") + File.separator + "my-private-key");
```

Comment:
In #17 I changed this to use a JUnit @Tempdir, which I think would be preferable, that way it gets cleaned up automagically by JUnit framework.

Reply:
Thanks, I did not know this annotation yet :thumbsup: It's definitively better.